### PR TITLE
Add metadata

### PR DESCRIPTION
**Pull request by spetrosi**
Add licensing and edit meta/main.yml.

### CODE REVIEW

**Comment by richm** [comment]
You still need the empty extra_requirements.txt files - if you want to get rid of them, you'll have to change tox-lsr to not attempt to use them

**Comment by spetrosi** [comment]
I have returned the files for now, I might require to use them in future.

File: meta/main.yml [hunk 1]
```diff
@@ -1,68 +1,14 @@
...
-  galaxy_tags: []
-  # List tags for your role here, one per line. A tag is a keyword that
-  # describes and categorizes the role. Users find roles by searching for tags.
-  # Be sure to remove the '[]' above, if you add tags to this list.
-  #
-  # NOTE: A tag is limited to a single word comprised of alphanumeric
-  #       characters. Maximum 20 tags per role.
-
+        - 8
+        - 7
```
Comment (richm) [inline meta/main.yml:11]: 7?  The README says RHEL 8 only.

**Comment by spetrosi** [comment]
I will keep the whole work within the #2 PR for easier reviews. I've cherry-picked commits from this PR to #2 and will close this PR now.

**Pull request closed without merging**